### DISCUSSION TITLE
TAK ICU

### DISCUSSION THREAD

**Discussion by vaderman1998** [bluenviron/mediamtx#3433] [Questions]
### Question

Not sure if anyone can help but im using mediamtx in my TAK server and when I go to stream using TAK ICU I get "muxing video failed(2)" Any help would be great

**Comment by aler9** [comment]
Hello, it's impossible to replicate the problem without further data. Reopen the question and provide

* server version
* server logs
* Tak ICU version
* step by step instructions on how to replicate the problem
